go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/google/go-querystring v1.1.0 // indirect
 	github.com/google/gofuzz v1.1.0 // indirect
-	github.com/google/uuid v1.1.2 // indirect
+	github.com/google/uuid v1.3.0 // indirect
```

Comment:
Why are google/uuid, x/net, and changes like that needed?

Reply:
those got updated when I updated the k8s.io/kubernetes